src/dynamic_batch_scheduler.cc

```diff
@@ -163,6 +163,11 @@ DynamicBatchScheduler::~DynamicBatchScheduler()
 Status
 DynamicBatchScheduler::Enqueue(std::unique_ptr<InferenceRequest>& request)
 {
+  if (stop_) {
+    return Status(
+        Status::Code::UNAVAILABLE,
+        "Scheduler has stopped accepting new inference");
```

Comment:
Perhaps say something indicating why... "Server is stopping, scheduler for model <xyz> has stopped accepting new inference requests: